code-challenges.test.js

```diff
@@ -20,37 +20,109 @@
 
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
-
-
-// b) Create the function that makes the test pass.
-
+//Psuedo code: what im going to do first is that i have to "describe" and add my function.
+//After is the "it" or test and here I write what i want in return and what i want is an array withthe lengthof the given variable in Fibonacci sequence.
+
+ describe("fibonacciArr", () => {
+  it("return an array with the length of the given variable in Fibonacci sequence", () =>{
+   expect(fibSeq(6)).toEqual([1,1,2,3,5,8]);
+   expect(fibSeq(10)).toEqual([1,1,2,3,5,8,13,21,34,55]);
+   })
+ })
+// // My test fail because my input is not defined so now i have to make tos test pass, what im going to do first is that i have to declare a variable named input.
+// //What i have to do next is create a for loop to make it go arounf the Fibonacci sequence to loop around the lengthof the array. I want the first length at 1 and 1.
+// // I want to add the number to the next 2 numbers.
+//Firts im going to declare my variable input
+// // b) Create the function that makes the test pass.
+ const fibSeq = (array) => {
+   let first = [1,1]
+   for(let i=2; i<array; i++){
+     first.push((first[(first.length -1)]) + (first[(first.length -2)]))
+   }
+   return first
+ }
+// Its green yay!!!
```

Comment:
Woohoo! Great job. Your code and your documentation is looking really wonderful!